osu.Game/Overlays/Profile/Header/Components/RankGraph.cs

```diff
@@ -66,18 +64,25 @@ private void load(OsuColour colours)
             graph.LineColour = colours.Yellow;
         }
 
-        private void userChanged(ValueChangedEvent<User> e)
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            Statistics.BindValueChanged(statistics => updateStatistics(statistics.NewValue));
```

Comment:
probably want to specify the second argument to force a state change, else the display won't be correct if the statistics are changed before the `RankGraph` is loaded.